craft_parts/packages/platform.py

```diff
@@ -15,24 +15,44 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 """Helpers to determine the repository for the platform."""
-from typing import Optional
+from typing import List, Optional
 
 from craft_parts import errors
 from craft_parts.utils import os_utils
 
 _DEB_BASED_PLATFORM = ["ubuntu", "debian", "elementary OS", "elementary", "neon"]
+_RPM_BASED_PLATFORM = ["fedora", "centos"]
```

Comment:
Fedora today only support DNF, so I would remove it for now

Reply:
> > Looks good, but I would change RPM to DNF,
> 
> You mean "change RPM to YUM", right?

Confirmed through chat. Rename done.